packages/synapse-react-client/src/assets/icons/NoSearchResults.tsx

```diff
@@ -2,56 +2,13 @@ import { Box, BoxProps } from '@mui/material'
-        fill="white"
-      />
-      <path
-        d="M355.177 295.255H191.302C190.21 295.253 189.162 294.819 188.39 294.046C187.617 293.274 187.182 292.226 187.181 291.134V94.7752C187.182 93.6825 187.617 92.635 188.39 91.8624C189.162 91.0899 190.21 90.6553 191.302 90.6541H355.177C356.269 90.6553 357.317 91.0899 358.089 91.8625C358.862 92.6351 359.296 93.6826 359.298 94.7752V291.134C359.296 292.226 358.862 293.274 358.089 294.046C357.317 294.819 356.269 295.253 355.177 295.255Z"
-        fill="#F1F3F5"
-      />
-      <path
-        d="M279.784 80.9573H177.727C175.542 80.9604 173.447 81.8297 171.902 83.3748C170.357 84.9198 169.487 87.0145 169.484 89.1995V286.915L170.454 286.62V89.1995C170.456 87.2714 171.223 85.423 172.587 84.0596C173.95 82.6963 175.799 81.9293 177.727 81.9269H280.085L279.784 80.9573ZM368.752 80.9573H177.727C175.542 80.9604 173.447 81.8297 171.902 83.3748C170.357 84.9198 169.487 87.0145 169.484 89.1995V311.254C169.487 313.439 170.357 315.534 171.902 317.079C173.447 318.624 175.542 319.493 177.727 319.496H368.752C370.937 319.493 373.031 318.624 374.576 317.079C376.121 315.534 376.991 313.439 376.994 311.254V89.1995C376.991 87.0145 376.121 84.9198 374.576 83.3748C373.031 81.8297 370.937 80.9604 368.752 80.9573ZM376.024 311.254C376.022 313.182 375.255 315.031 373.892 316.394C372.528 317.757 370.68 318.524 368.752 318.527H177.727C175.799 318.524 173.95 317.757 172.587 316.394C171.223 315.031 170.456 313.182 170.454 311.254V89.1995C170.456 87.2714 171.223 85.423 172.587 84.0596C173.95 82.6963 175.799 81.9293 177.727 81.9269H368.752C370.68 81.9293 372.528 82.6963 373.892 84.0596C375.255 85.423 376.022 87.2714 376.024 89.1995V311.254Z"
-        fill="#DFE2E6"
-      />
-      <path
-        d="M317.844 102.29H228.635C227.478 102.289 226.369 101.829 225.55 101.011C224.732 100.192 224.272 99.0834 224.271 97.9265V76.5937C224.272 75.4369 224.732 74.3277 225.55 73.5097C226.369 72.6917 227.478 72.2315 228.635 72.2302H317.844C319.001 72.2315 320.11 72.6917 320.928 73.5097C321.746 74.3277 322.206 75.4369 322.208 76.5937V97.9265C322.206 99.0834 321.746 100.192 320.928 101.011C320.11 101.829 319.001 102.289 317.844 102.29Z"
-        fill="#AEB5BC"
-      />
-      <path
-        d="M273.239 73.6848C278.595 73.6848 282.936 69.3434 282.936 63.9881C282.936 58.6327 278.595 54.2914 273.239 54.2914C267.884 54.2914 263.542 58.6327 263.542 63.9881C263.542 69.3434 267.884 73.6848 273.239 73.6848Z"
-        fill="#AEB5BC"
-      />
-      <path
-        d="M273.24 69.8943C276.502 69.8943 279.146 67.25 279.146 63.9881C279.146 60.7262 276.502 58.0819 273.24 58.0819C269.978 58.0819 267.333 60.7262 267.333 63.9881C267.333 67.25 269.978 69.8943 273.24 69.8943Z"
-        fill="white"
-      />
-    </svg>
-  </Box>
+  <Box
+    {...props}
+    component={'img'}
+    src={
+      'https://s3.us-east-1.amazonaws.com/static.synapse.org/images/NoSearchResults.svg'
```

Comment:
I put this file in S3 to simplify getting it in SWC. We aren't re-theming it so there's no reason for the SVG data to live in a React component.